src/getntbe.f

```diff
@@ -1,36 +1,24 @@
 C> @file
 C> @author ATOR @date 2007-01-19
 	
-C> THIS SUBROUTINE GETS THE FIRST LINE OF THE NEXT ENTRY IN
-C>   THE SPECIFIED ASCII MASTER TABLE B OR MASTER TABLE D FILE.  THIS
-C>   LINE CONTAINS, AMONG OTHER THINGS, THE FXY NUMBER CORRESPONDING TO
-C>   THIS ENTRY.
+C> This subroutine gets the first line of the next entry in
+C> the specified ascii master table b or master table d file. This
+C> line contains, among other things, the fxy number corresponding to
+C> this entry.
 C>
-C> PROGRAM HISTORY LOG:
-C> 2007-01-19  J. ATOR    -- ORIGINAL AUTHOR
```

Comment:
@jbathegit I am removing these PROGRAM HISTORY LOG sections when they contain nothing but the initial creation of the subprogram. That's already captured in the author and date tags, so no need to repeat it.

Whether we should continue updating these history logs is unclear. This is what github does already. Do we need to continue doing it? That question has been asked and I don't have any strong answer... Do you?